src/psycopmlutils/feature_describer/feature_describer.py

```diff
@@ -103,24 +111,25 @@ def generate_feature_description_row(
         Dict: Dictionary with feature description.
     """
 
-    d = {}
-
-    d["Predictor df"] = predictor_dict["predictor_df"]
-    d["Lookbehind days"] = predictor_dict["lookbehind_days"]
-    d["Resolve multiple strategy"] = predictor_dict["resolve_multiple"]
-
-    d["Fallback strategy"] = predictor_dict["fallback"]
-
-    d["Proportion using fallback"] = get_value_proportion(series=series, value=np.nan)
-
-    d["Mean"] = round(series.mean(), 2)
+    d = {
+        "Predictor df": predictor_dict["predictor_df"],
+        "Lookbehind days": predictor_dict["lookbehind_days"],
+        "Resolve multiple": predictor_dict["resolve_multiple"],
+        "N unique": series.nunique(),
+        "Fallback strategy": predictor_dict["fallback"],
+        "Proportion missing": series.isna().mean(),
+        "Mean": round(series.mean(), 2),
+        "Histogram": create_unicode_hist(series),
+        "Proportion using fallback": get_value_proportion(
+            series,
+            predictor_dict["fallback"],
+        ),
+    }
```

Comment:
This function has very large overlap with `generate_column_description` now. Can we refactor?

Reply:
This one requires predictor df, so we won't merge the two (as dicussed yesterday)